src/Apps/Legion/Components/TypeIcon/TypeIcon.styles.ts

```diff
@@ -0,0 +1,26 @@
+import { ITypeIconStyles } from './TypeIcon.types';
+import {
+    IProcessedStyleSet,
+    mergeStyleSets,
+    memoizeFunction
+} from '@fluentui/react';
+
+export const getStyles = memoizeFunction(
+    (color: string): IProcessedStyleSet<ITypeIconStyles> => {
+        return mergeStyleSets({
+            root: {
+                display: 'inline-flex',
+                width: 32,
+                height: 32,
+                borderRadius: 16,
+                backgroundColor: `#${color}`,
+                alignItems: 'center',
+                justifyContent: 'center',
+                marginRight: 8
+            },
+            icon: {
+                color: 'white'
```

Comment:
this is intentional since the icon color is not theme aware?

Reply:
yes